README.md

```diff
@@ -80,7 +80,7 @@ The container runs via `docker_scripts/entrypoint.sh`, which
    $ docker-compose exec django python manage.py loaddata --app oh_staff_ui seed-data
    #
    # Load full set of item data
-   $ docker-compose exec django python manage.py import_projectitems oh-projectitems-export-3.csv
+   $ docker-compose exec django python manage.py import_projectitems project-items-export.tsv
```

Comment:
This example should have the path in front of the filename: `export_scripts/project-items-export.tsv`

Reply:
Updated `README.md` to properly reflect path